File: DSA PRACTICE/LECTURE 22/ToLowerCase.cpp
// PROGRAM 03: Character NOT Case Sensitive 'S'='s'

#include <iostream>
using namespace std;

char toLowerCase(char ch)
{
    if(ch>='a' && ch<='z')
    {
        return ch;
    }
    else if(ch>='A' && ch<='B')
    {
        char temp=ch - 'A'+'a';
        return temp;
    }
}

int main()
{
    cout<<"Character b: "<<toLowerCase('b')<<endl;
    cout<<"Character B: "<<toLowerCase('B')<<endl;


    return 0;

}


/*
Character b: b
Character B: b
*/

File: DSA PRACTICE/LECTURE 22/ToLowerCase.java
// PROGRAM 03: Upper to Lower case 'S'='s'

class ToLowerCase
{
    public static void main(String[] args) 
    {
       System.out.print("Character b: "+toLowerCase('b')); 
       System.out.print("Character B: "+toLowerCase('b')); 
    }

    public static char toLowerCase(char ch)
    {
        int temp=0;
        if(ch>='a' && ch<='z')
        {
            return ch;
        }
        else if(ch>='A' && ch<='Z')
        {
            temp=((int)ch )- 'A'+'a';
        }
       return (char)temp; 
    }
}

/*
Character b: b
Character B: b
*/


/*Lower to Upper
public static char toUpperCase(char ch)
    {
        int temp=0;
        if(ch>='A' && ch<='Z')
        {
            return ch;
        }
        else if(ch>='a' && ch<='z')
        {
            temp=((int)ch )- 'a'+'A';
        }
       return (char)temp; 
    }
*/